src/axolotl/integrations/nvfp4/swap.py

```
@@ -0,0 +1,331 @@
+        convert_to_nvfp4_training,
+    )
+
+    recipe = NVFP4Recipe(
+        stochastic_rounding=nvfp4.stochastic_rounding,
+        hadamard=nvfp4.hadamard,
+    )
+    skip_first, skip_last = _resolve_keep_hp_counts(nvfp4, model)
+    # lm_head/embeddings are handled by the keyword path, not as body linears.
+    body_exclude = ("lm_head", "embed_tokens") + _block_exclusions(
+        model, skip_first, skip_last
+    )
```

Ruff (0.15.20)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)